src/zenml/cli/stack.py

```diff
@@ -1944,3 +1944,79 @@ def query_region(
         service_connector_index=service_connector_index,
         service_connector_resource_id=service_connector_resource_id,
     )
+
+
+@stack.command(
+    name="export-requirements", help="Export the stack requirements."
+)
+@click.argument(
+    "stack_name_or_id",
+    type=click.STRING,
+    required=False,
+)
+@click.option(
+    "--output-file",
+    "-o",
+    "output_file",
+    type=str,
+    required=False,
+    help="File to which to export the stack requirements. If not "
+    "provided, the requirements will be printed to stdout instead.",
+)
+@click.option(
+    "--overwrite",
+    "-ov",
+    "overwrite",
+    type=bool,
+    required=False,
+    is_flag=True,
+    help="Overwrite the output file if it already exists. This option is "
+    "only valid if the output file is provided.",
+)
+def export_requirements(
+    stack_name_or_id: Optional[str] = None,
+    output_file: Optional[str] = None,
+    overwrite: bool = False,
+) -> None:
+    """Exports stack requirements so they can be installed using pip.
+
+    Args:
+        stack_name_or_id: Stack name or ID. If not given, the active stack will
+            be used.
+        output_file: Optional path to the requirements output file.
+        overwrite: Overwrite the output file if it already exists. This option
+            is only valid if the output file is provided.
+    """
+    from zenml.stack import Stack
+
+    try:
+        stack_model: "StackResponse" = Client().get_stack(
+            name_id_or_prefix=stack_name_or_id
+        )
+    except KeyError as err:
+        cli_utils.error(str(err))
+
+    stack_ = Stack.from_model(stack_model)
+    requirements = list(stack_.requirements())
```

Comment:
As far as I know, the `stack_.requirements` call here combines the requirements of this stack's components, which combines the requirements of its integrations. However in order to be able to do this, you initialize the `Stack` object.

Since you have the `stack_model` here, I think we can avoid initializing the `Stack` object here. Each component within this `stack_model` should already have an `integration` attribute which can help you get the proper requirements. Moreover, this way we could support the main use-case that this CLI function would help with: 

- I see a stack -> I want to use the stack -> I need to install the requirements for it first -> I do export-requirements and install it -> I use the stack.

Otherwise, you need to have them installed before exporting the requirements.

Reply:
Yeah I was just too lazy to do this, I now used the util method that we already have for this :)

Reply:
Love it! It enables the use-case that I mentioned, which is very nice 👍